lib/atom_linter.dart

```diff
@@ -66,7 +67,26 @@ class LinterService extends ProxyHolder {
 
   void setMessages(LinterProvider provider, List<LintMessage> messages) {
     var list = messages.map((m) => m.toMap()).toList();
-    _linter.invoke('setMessages', list);
+    _linter.invoke('setAllMessages', list);
+  }
+}
+
+class LintSolution {
+
+  final String title;
+  final num priority;
+  final Rn position;
+  Function apply;
```

Comment:
Perhaps use a typedef instead of `Function`? Something like: `typedef void VoidFunc();`